uint8 UNINIT=0
uint8 READY=1
uint8 ACTIVE=2
uint8 COMPLETE=3
uint8 FAILED=4
uint8 STANDBY=5
uint8 status
string text
